src/odemis/driver/test/actuator_test.py

```diff
@@ -1591,6 +1591,103 @@ def test_cancel_reference(self):
         # Some axes could be referenced during this time but not all
         self.assertFalse(all(stage.referenced.value.values()))
 
+class TestLinkedAxesActuator(unittest.TestCase):
+
+    def setUp(self):
+        # Construct the underlying sem stage
+        # Using TMCLController to correctly simulate movement
+        self.dep_stage = tmcm.TMCLController(name="dep_stage", role="sem_stage",
+                                             port="/dev/fake3", axes=["x", "y", "z"],
+                                             ustepsize=[STEP_SIZE, STEP_SIZE, STEP_SIZE],
+                                             rng=[[-6e-3, 6e-3], [-6e-3, 6e-3], [-6e-3, 6e-3]],
+                                             refproc="Standard")
+
+        # Create Linked axes stage from the dependant stage
+        self.linked_axes = LinkedAxesActuator("Linked Axes", "stage", dependencies={"stage": self.dep_stage}, )
+        # update metadata
+        self.linked_axes.updateMetadata({model.MD_POS_COR: [0, 0, 0]})
+        # cls.linked_axes.updateMetadata({model.MD_CALIB: [[1, 0], [0, 0.707], [0, 0.707]]})
+        self.linked_axes.updateMetadata({model.MD_CALIB: [[1, 0], [0, 1], [0, 0]]})
```

Comment:
With these identity calibration, an extra thing that you can check is that the position of the X and Y dependent axes are the same as the wrapped X and Y.